README.md

```diff
@@ -1,30 +1,48 @@
-# jiant v2 (Alpha)
-[![Generic badge](https://img.shields.io/badge/version-2.0.dev0-blue.svg)](https://shields.io/)
-[![codecov](https://codecov.io/gh/jiant-dev/jiant/branch/master/graph/badge.svg)](https://codecov.io/gh/jiant-dev/jiant)
+<div align="center">
+
+# `jiant`
+**The multitask and transfer learning toolkit for natural language processing research**
+
+[![Generic badge](https://img.shields.io/github/v/release/nyu-mll/jiant)](https://shields.io/)
+[![codecov](https://codecov.io/gh/nyu-mll/jiant/branch/master/graph/badge.svg)](https://codecov.io/gh/nyu-mll/jiant)
+[![CircleCI](https://circleci.com/gh/nyu-mll/jiant/tree/master.svg?style=shield)](https://circleci.com/gh/nyu-mll/jiant/tree/master)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
-`jiant` is a software toolkit for natural language processing research, designed to facilitate work on multitask learning and transfer learning for sentence understanding tasks. 
+</div>
+
+**Why should I use `jiant`?**
```

Comment:
@zphang I'm looking for a concise statement about why people should use `jiant` over `transformers` for training/evaluation. Phil mentioned that you found the "seams" in `transformers` that caused problems with research. Do you think you can capture this in a bullet point?